pkg/promlib/models/scope_test.go

```diff
@@ -27,6 +27,12 @@ func TestApplyQueryFiltersAndGroupBy_Filters(t *testing.T) {
 			expected:  `http_requests_total{job="prometheus"}`,
 			expectErr: false,
 		},
+		{
```

Comment:
Could you move these into another set of tests of UTF testing, thinking we won't really be looking to replicate every case here going forward.

Reply:
https://github.com/grafana/grafana/pull/99015/commits/7701cf69e836fe20897732dd7b0b36913845461d